infra/core/database/postgresql/flexibleserver.bicep

```diff
@@ -75,5 +75,4 @@ resource keyVault 'Microsoft.KeyVault/vaults@2022-07-01' existing = {
   name: keyVaultName
 }
 
-output POSTGRES_SERVER_NAME string = postgresServer.name
```

Comment:
why these changes in this file?

Reply:
Uhhh.... I just did git rebase from the main.... let me remove